src/main.cpp

```diff
@@ -7170,7 +7170,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             return false;
         }
 
-		if (pfrom->nVersion < 180322 & !fTestNet && pindexBest->nHeight > 855000)
+		if (pfrom->nVersion < 180322 && !fTestNet && pindexBest->nHeight > 855000)

```

Comment:
result of first '<' is 0/1 and result of '!' is 0/1 so the '&' does the same thing to it
then it goes to '&&' as boolean
which means that the two expressions are equivalent